### PR TITLE
Add blog post announcing demos for Pulp 3 GA

### PR DESCRIPTION
**Pull request by CodeHeeler**

### CODE REVIEW

File: _posts/2020-01-20-pulp-3-GA-demos.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Pulp 3.0 GA Demos
+author: Dana Walker
+tags:
+  - 3.0
+---
+To go along with Pulp 3.0 GA, we have prepared a playlist of demos to cover features in pulpcore as
+well as various features from some of our plugins.
```
Comment (daviddavis) [inline _posts/2020-01-20-pulp-3-GA-demos.md]: Just pulpcore (for now). We can announce the other plugin playlists when they are ready.

File: _posts/2020-01-20-pulp-3-GA-demos.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+We will continue adding content to this playlist, so be sure to check it out and let us know if you
```
Comment (daviddavis) [inline _posts/2020-01-20-pulp-3-GA-demos.md]: I think the playlist covers all the basic features of Pulpcore so I don't think we'll continue adding to it.

File: _posts/2020-01-20-pulp-3-GA-demos.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+find it useful.  Also, if there is any content you don't see but would like a demo to cover, please
+reach out to us with that request.
```
Comment (daviddavis) [inline _posts/2020-01-20-pulp-3-GA-demos.md]: :+1:

**Comment by daviddavis** [comment]
Any chance you can link or embed the playlist?

**Review by daviddavis — APPROVED** [review]

File: _posts/2020-01-20-pulp-3-GA-demos.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+  - 3.0
+---
+To go along with Pulp 3.0 GA, we have prepared a playlist of demos to cover features in pulpcore on
+the [Pulp YouTube Channel](https://www.youtube.com/pulpproject).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PLwm8_O6oKSS0C3zlo8gFkPqXpMkAI7rCD" frameborder="0" allowfullscreen></iframe>
+
+Check it out and let us know if you find it useful.  Also, if there is any content you don't see but
+would like a demo to cover, please reach out to us with that request.
```
Comment (bmbouter) [inline _posts/2020-01-20-pulp-3-GA-demos.md]: Can we identify how they will reach out? I think via pulp-list@redhat.com is probably what I would recommend. Maybe link to https://www.redhat.com/mailman/listinfo/pulp-list

**Review by bmbouter — APPROVED** [review]
This looks good to me. @CodeHeeler are we ready to merge? If so, please do.

**Pull request merged**